.yamato/project-standards.yml

```diff
@@ -9,7 +9,12 @@ standards_{{ projects.first.name }}:
   commands:
     - dotnet --version
     - dotnet format --version
-    - pip install unity-downloader-cli --upgrade --index-url https://artifactory.prd.it.unity3d.com/artifactory/api/pypi/pypi/simple
+    - sudo add-apt-repository -y ppa:deadsnakes/ppa
+    - sudo apt update
+    - sudo apt install -y python3.9
```

Comment:
I need a second PR to upgrade the python version for all the other jobs as well, but I would need find out the installation command per OS (see other jobs image...)
In the interest of time and unblocking the release for U6, I am only upgrading this job to use python 3.